README.md

```diff
@@ -16,6 +16,7 @@ Don't forget to:
 Total deals: 0
 
 - [👨‍💻 Developer Tools ](#-developer-tools)
+	- [🛠️ APIs (1)](#-apis)
```

Comment:
Please move this section below the `Tools & Services` as it is first come first served. Thanks for your understanding

Reply:
Just changed. Thanks!